build/config/PublishData.json

```diff
@@ -28,6 +28,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.9p1"]
         },
+        "dev16.0.x-vs-deps": {
```

Comment:
I suspect this part of the change needs to happen in the `dev16.0.x-vs-deps` branch?

Reply:
@Pilchie: no, the publishing tools always reads it out of the master branch.